api/http/refresh.go

```diff
@@ -17,22 +17,20 @@ type RefreshHandler struct {
 
 // ServeHTTP refreshes a given token.
 func (service RefreshHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	account := &api.Account{}
 
 	// Valid token and audience
 	_, id, err := service.Token.CheckToken(r)
```

Comment:
In other handlers you've removed this check entirely since it was already handled by the http.JWTHandler. Does it make sense to make that consistent here, and add http.JWTHandler to where `/refresh` is set up in `main.go`? Would that make all the top-level authn happen in that one place?

Reply:
I like that idea. Refresh is configured without the middleware currently, should be easy to fix.